it/apps/src/main/content/jcr_root/apps/forms-core-components-it/samplepages/loadunloadaf/v1/loadunloadaf/loadunloadaf.html

```diff
@@ -0,0 +1,43 @@
+<div class="custom-embed">
+        <script>
+            
+           function loadform(btn) {
+             var formPath = btn.getAttribute('data-form-path')
+             loadAdaptiveForm({
+               formPath: formPath,
+               CSS_Selector: '.customafsection'
+             })
+           }
+
+           var loadAdaptiveForm = function(options) {
+             var path = options.formPath + '.html?wcmmode=disabled';
```

Comment:
how will this work if one needs to only show the form container (ie) without header/footer. In those case, we would have to also include the runtime library in the main page. Can you write a test case for this and also update the documentation ?